src/App.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit <code>src/App.tsx</code> and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
```

Comment:
- [ ] You have done great so far. :+1:  But there are some issues you need to fix before moving to the next milestone. :smile: :small_airplane: 
- [ ] Looking at the video below, you can see that when I start the app, the browser is black without displaying the default react logo and text. I'll suggest you look into this as your app is meant to run normally after the setup. :+1: :100: 
- [ ] Also it seems you are using typescript in this project. this is not part of the requirement, I'll suggest you revert your code to using plane vanilla JS. :+1: 

- - Video reference:

https://github.com/anyars-encarta/math-magician/assets/107632780/2f01ac59-be30-4919-af55-3de013877e58

Reply:
Looking into it